infrastructure/server/oidc_idp_bcsc.tf

```diff
@@ -16,25 +19,24 @@ resource "aws_cognito_identity_provider" "dev_bcsc_oidc_provider" {
     client_secret             = var.dev_oidc_bcsc_idp_client_secret
     oidc_issuer               = var.dev_bcsc_oidc_idp_issuer
     attributes_request_method = "GET"
-    authorize_url = "https://idtest.gov.bc.ca/login/oidc/authorize"
-    token_url = "${aws_api_gateway_deployment.fam_api_gateway_deployment.invoke_url}/bcsc/token/dev"
-    attributes_url = "${var.use_override_proxy_endpoints ?  var.dev_override_bcsc_userinfo_proxy_endpoint : local.dev_local_bcsc_userinfo_proxy_endpoint}"
-
-    jwks_uri = "https://idtest.gov.bc.ca/oauth2/jwk.json"
+    authorize_url             = "https://idtest.gov.bc.ca/login/oidc/authorize"
+    token_url                 = "${var.use_override_proxy_endpoints ? var.dev_override_bcsc_token_proxy_endpoint : local.dev_local_bcsc_token_proxy_endpoint}"
```

Comment:
so we have a `use_override_proxy_endpoints` variable, which will only be used in AWS DEV environment, where we can set the token_url and attribute_url (userinfo url) to use the production api url 
![image](https://github.com/bcgov/nr-forests-access-management/assets/23131735/3977551b-38f6-4452-ae07-4e4203f83636)